src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -1031,14 +1031,12 @@ ClassPathEntry* find_first_module_cpe(ModuleEntry* mod_entry,
 // Search either the patch-module or exploded build entries for class.
 ClassFileStream* ClassLoader::search_module_entries(JavaThread* current,
                                                     const GrowableArray<ModuleClassPathList*>* const module_list,
-                                                    const char* const class_name,
+                                                    Symbol* name,
```

Comment:
AFAICS `name` is never actually used. The class name was only used to get the package entry, which we already have.

The comment on L1031 also needs updating as no class is involved.

Reply:
Good catch!